website/content/docs/ecs/get-started/production-installation.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: Production Installation - AWS ECS
+description: >-
+  Production Installation of the Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Production Installation
+
+For a production-ready installation of Consul on ECS, you will need to make sure that the cluster is secured.
+Secure Consul cluster should include the following:
+
+1. [TLS Encryption](/docs/security/encryption#rpc-encryption-with-tls) for RPC communication between Consul clients and servers.
+1. [Gossip Encryption](/docs/security/encryption#gossip-encryption) for encrypting gossip traffic.
+1. [Access Control (ACLs)](/docs/security/acl) for authentication and authorization for Consul clients and services on the mesh.
+
+-> **NOTE:** This page assumes that you have already configured your Consul server with the above features.
+
+## Deploy ACL Controller
+
+Before deploying your service, you will need to deploy the [ACL controller](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/acl-controller) so that it can provision the necessary tokens
+for tasks on the service mesh. To learn more about the ACL Controller, please see [Automatic ACL Token Provisioning](/docs/ecs/architecture#automatic-acl-token-provisioning).
+
+To deploy the controller, you will first need store an ACL token with `acl:write` privileges
+and CA certificate for the Consul server in AWS Secrets Manager.
+
+```hcl
+resource "aws_secretsmanager_secret" "bootstrap_token" {
+  name  = "bootstrap-token"
+}
+
+resource "aws_secretsmanager_secret_version" "bootstrap_token" {
+  secret_id     = aws_secretsmanager_secret.bootstrap_token[count.index].id
+  secret_string = random_uuid.bootstrap_token[count.index].result
```

Comment:
I think we can remove `count.index` in this example.

Reply:
lol thank you! yes this was a bad copy-pasta 😄